website/docs/d/mks_feature_gates_v1.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "selectel"
+page_title: "Selectel: selectel_mks_feature_gates_v1"
+sidebar_current: "docs-selectel-datasource-mks-feature-gates-v1"
+description: |-
+  Get information on Selectel MKS available feature gates.
+---
+
+# selectel\_mks\_feature_gates_v1
+
+Use this data source to get available feature-gates within Selectel MKS API Service.
+
+## Example Usage
+
+```hcl
+resource "selectel_vpc_project_v2" "project_1" {
+  auto_quotas = true
+}
+
+data "selectel_mks_feature_gates_v1" "fg" {
+  project_id = "${selectel_vpc_project_v2.project_1.id}"
+  region = "ru-3"
+  filter {
+    kube_version = "1.22."
```

Comment:
Patch version is missed. Please add it or keep just a minor version without dot in the end.